sys/include/fido2/ctap/ctap_mem.h

```diff
@@ -41,28 +41,9 @@ extern "C" {
 /** @} */
 
 /**
- * @brief First flash page to store data in
- *
- * @note This can corrupt firmware if CTAP_FLASH_START_PAGE is set to a
- * flash page containing firmware. Therefore make sure that CTAP_FLASH_START_PAGE
- * is located after the firmware.
- */
-#if defined(CONFIG_FIDO2_CTAP_FLASH_START_PAGE) && \
-    (CONFIG_FIDO2_CTAP_FLASH_START_PAGE >= 0)
-#define CTAP_FLASH_START_PAGE CONFIG_FIDO2_CTAP_FLASH_START_PAGE
-#else
-#define CTAP_FLASH_START_PAGE (FLASHPAGE_NUMOF - 4)
-#endif
-
-/**
- * @brief Start page for storing resident keys
- */
-#define CTAP_FLASH_RK_START_PAGE CTAP_FLASH_START_PAGE
-
-/**
- * @brief Page for storing authenticator state information
+ * @brief   Default amount of flashpages to use
  */
-#define CTAP_FLASH_STATE_PAGE CTAP_FLASH_RK_START_PAGE - 1
+#define CTAP_FLASH_DEFAULT_AMOUNT_FLASHPAGES 4
```

Comment:
The default value of `FIDO2_CTAP_FLASH_NUM_FLASHPAGES` is already 4. Do we need the additional `#define`